librecad/src/ui/forms/qg_commandwidget.cpp

```diff
@@ -155,7 +157,48 @@ void QG_CommandWidget::tabPressed() {
             leCommand->setText(reducedChoice.first());
         }
         else if (reducedChoice.count()>0) {
+            QString longestString = "";
+            QString shortestString = "";
+            int lengthShortestString(0);
+            int lengthReducedChoice = reducedChoice.count();
+
+            // Finding which is the longest string
```

Comment:
a common string to propose is a substring of the shortest, and the longest string can never be proposed.

If it's the case, we don't have to search for the longest.